x-pack/test/functional/apps/api_keys/home_page.ts

```diff
@@ -13,11 +13,26 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
   const log = getService('log');
   const security = getService('security');
   const testSubjects = getService('testSubjects');
+  const es = getService('es');
   const find = getService('find');
   const browser = getService('browser');
 
+  async function clearAllApiKeys() {
+    const existingKeys = await es.security.queryApiKeys();
+    if (existingKeys.count > 0) {
+      Promise.all(
+        await existingKeys.api_keys.map(async (key) => {
+          await es.security.invalidateApiKey({ ids: [key.id] });
+        })
+      );
+    } else {
+      log.debug('No API keys to delete.');
+    }
+  }
+
   describe('Home page', function () {
     before(async () => {
+      await clearAllApiKeys();
```

Comment:
I remember @LeeDr mentioning it should be added into `after` of the test suite responsible for adding these keys.
Also, looking at the code locally, I wonder why is this function is placed in test directly? I think it is better to put it in `common_page.ts` so we can re-use it later in other suites

Reply:
I would think that this is something that we would maybe put in a helper file at most. Something like `Pageobjects.common.sleep()` should be in `common_page.ts` because no matter where you are in Kibana, you may need a few seconds of delay. But `clearAllApiKeys()` is performing an action specifically in the API keys app. We would never be on the `visualize` page and then need to clear API keys.